### ISSUE TITLE
improve mobile responsivity

### ISSUE THREAD

**Issue by bagage** [nrenner/brouter-web#296]
I tweaked some files on my fork to improve web experience via PWA (progressive web app).

I'll submit a PR for that, I think we could do a TWA too quite easily so that brouterweb can be installed via app store (play store or fdroid or such). It will require #286.

I think we could improve UI in a few areas:
- [x] put everything on a single line at the bottom
- [x] put everything on a single column on the left side (or maybe remove some buttons on mobile look zoom buttons)
- [ ] improve the search icon quality that looks blurry on mobile (need to update the library but blocked by upstream issue yet https://github.com/perliedman/leaflet-control-geocoder/issues/324)
- [x] somehow improve the navbar which is not very convenient on mobile *won't do that here, it is a rather separate issue*
- [x] improve profile/alternative selection on mobile (you need double click to select an item but it seems to take effect on single click too - but related to previous point)
- [x] make modals fullscreen?
- [x] remove fullscreen button from profile layers selection on small screens
- [x] maybe more…

For now it looks like this:

![Screenshot_20200521-134854_Firefox_Preview](https://user-images.githubusercontent.com/1451988/82556706-8fd28a80-9b6a-11ea-987b-90f5f898f4ae.png)

**Comment by nrenner** [comment]
Yes, going full screen would be nice.

Not quite clear to me: is the icon required for your PR or a potential future TWA?

**Comment by bagage** [comment]
Not strictly required, a placeholer icon is fine too. And twa is the last item to do, once everything else is fixed.
